rules/linux/execution_abnormal_process_id_file_created.toml

```diff
@@ -0,0 +1,73 @@
+[metadata]
+creation_date = "2022/05/11"
+maturity = "production"
+updated_date = "2022/05/11"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation of a Process ID (PID), lock or reboot file created in temporary file storage paradigm (tmpfs) directory
+/var/run. On Linux, the PID files typically hold the process ID to track previous copies running and manage other tasks.
+Certain Linux malware use the /var/run directory for holding data, executables and other tasks, disguising itself or
+these files as legitimate PID files.
+"""
+false_positives = [
+    """
+    False-Positives (FP) can appear if the PID file is legitimate and holding a process ID as intended. To
+    differentiate, if the PID file is an executable or larger than 10 bytes, it should be ruled suspicious.
```

Comment:
We can add an OSquery in the investigation guide to pull file size / other data to help troubleshoot

Reply:
Added verbiage about Osquery to both.